extensions/wikia/Email/EmailImageHelper.class.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Email;
+
+/**
+ * Class ImageHelper
+ *
+ * @package Email
+ */
+class ImageHelper {
+
+	// Cache icon information for a day
+	const ICON_CACHE_TTL = 86400;
+
+	static public $icons = [
+		'Wikia',
+		'Comics',
+		'Movies',
+		'Lifestyle',
+		'Music',
+		'Books',
+		'TV',
+		'Twitter',
+		'Facebook',
+		'YouTube',
+	];
+
+	/**
+	 * Get URL, size and name information about icons created in the newsletter wikia for our emails
+	 *
+	 * @return array
+	 */
+	public static function getIconInfo() {
+		$info = [];
+
+		foreach ( self::$icons as $name ) {
+			$fileInfo = self::getFileInfo( $name );
+			$info[$name] = $fileInfo;
+		}
+
+		return $info;
+	}
+
+	protected static function getFileInfo( $name ) {
+		$key = self::getFileKey( $name );
+		$memc = \F::app()->wg->Memc;
+
+		$info = $memc->get( $key );
```

Comment:
What do you think about using WikiaDataAccess for caching? 
https://github.com/Wikia/app/blob/dev/includes/wikia/WikiaDataAccess.class.php#L52